.install/linux_install_cmake.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+wget https://github.com/Kitware/CMake/releases/download/v3.24.3/cmake-3.24.3-linux-x86_64.sh 
+chmod u+x ./cmake-3.24.3-linux-x86_64.sh
```

Comment:
Probably want to variablize x86-64 to be `uname -m|sed -e 's/_/-/'` for when the 'default flow' is one on each platform. Yes - this could be premature optimization.